src/components/ProfileList/ProfileListItem.tsx

```diff
@@ -5,13 +5,16 @@ import React, { useEffect, useState } from 'react'
 import { JobSpecialization } from '@/components/ProfileList/profile-data'
 import ProfilePicture from '@/assets/images/ProfilePicture.png'
 import { ProfileModel } from '@/data/frontend/profile/types'
+import { AppRoutes } from '@/utils/routes'
+import { useRouter } from 'next/navigation'
 
 const cx = classNames.bind(styles)
 
 export const ProfileListItem: React.FC<{ data: ProfileModel }> = ({ data }) => {
   const [windowWidth, setWindowWidth] = useState(
     typeof window !== 'undefined' ? window.innerWidth : 1000,
   )
+  const router = useRouter()
 
   useEffect(() => {
     if (typeof window !== 'undefined') {
```

Comment:
**nitpick:** I know it's not your your change, but as long as this component is `use client` we can remove this `if (typeof window !== 'undefined')`, as window will never be undefined in client component